tests/tck-build-logic/src/main/java/org/graalvm/internal/tck/ScaffoldTask.java

```diff
@@ -28,7 +28,7 @@
  *
  * @author Moritz Halbritter
  */
-class ScaffoldTask extends DefaultTask {
+public class ScaffoldTask extends DefaultTask {
```

Comment:
This change was made because IDE reported following warning: `Access to 'ScaffoldTask' exceeds its access rights` in `tck-harness.gradle`.

Reply:
Does it actually cause a problem?

Reply:
I reverted the change since it is only a warning and doesn't cause a problem